### DISCUSSION TITLE
How many supabase projects do I need?

### DISCUSSION THREAD

**Discussion by Jamiecharris** [supabase/supabase#9712] [Questions]
Hi, I’m currently building an app. This app consists of 2 apps, the first app is customer facing and the second app is employee facing. The basic function of the app is a customer books (customer app), and then the employee receives the request and chooses wether to accept or not (employee app)
I want both apps to have separate authorisation and needs to share data between them (including storage data like images)
How should I setup Supabase, should I have 1 or 2 supabase projects, if 2 how do I share the data between them?

**Comment by dshukertjr** [comment]
In a setup like that, you want to contain everything in one database, hence you need one Supabase project. Using row level security, you can give consumer and employee different access permissions.
